.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x]
+        # See supported Node.js  release schedule at https://nodejs.org/en/about/releases/
```

Comment:
I don't think you need nodejs in the host since it's all running in the container

see for example this https://github.com/OpenUpSA/muni-portal-backend/blob/master/.github/workflows/test.yml#L20

Reply:
Done

Reply:
Removed